social/email/README.md

```diff
@@ -12,6 +12,8 @@ You will need valid email credentials for your email server.
 Install
 -------
 
+**Node.js v6 or newer is required.**
+
```

Comment:
Think we are going to need more explicit instructions how to upgrade - as it's a core node it's not installed  in ~/.node-red so need sudo etc... - or at least sudo to remove old then install this etc... - but I guess we can finness that before release.

Reply:
Any such upgrade guide is going to depend on how Node.js 4 was installed originally, and what platform the user is on.  If using `apt`, one would hope that upgrading would automatically remove the old version...